libraries/standard/mqtt/utest/mqtt_lightweight_utest.c

```diff
@@ -1687,6 +1687,17 @@ void test_MQTT_GetIncomingPacketTypeAndLength( void )
     TEST_ASSERT_EQUAL_INT( MQTT_PACKET_TYPE_PUBLISH, mqttPacket.type );
     TEST_ASSERT_EQUAL_INT( 128, mqttPacket.remainingLength );
 
+    /* Remaining length of 16384 needs 3 bytes. */
+    bufPtr = buffer;
+    buffer[ 0 ] = MQTT_PACKET_TYPE_PUBLISH;
+    buffer[ 1 ] = 0x80;
```

Comment:
Is 0x80 0x80 0x01 the encoding of remaining length 16384? If so, would you please mention that in the comment too?

Reply:
Yes. Sure, I'll update it here and for 128 above as well